### PR TITLE
docs(store): shorten desc of apps for cleaner view

### PR DESCRIPTION
**Pull request by JigSawFr**
This PR aim to give users a cleaner view of app in store and installed.
All ```short_desc``` of apps were rewritten to be cleaner, more precise and comprehensible.
Versions were also bumped to be showed on user interface,
Otherwise, many apps will not be updated due to the fact version is outdated or using latest tag.

Goal: See in 1 look what an app is for.

A screenshot:
image

### CODE REVIEW

File: apps/mylar3/config.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "$schema": "../schema.json",
-  "name": "Mylar3",
+  "name": "Mylar (3)",
```
Comment (meienberger) [inline apps/mylar3/config.json:3]: I think this should stay `Mylar3` as it is literally the name of the app (based on python3)

File: apps/paperless-ngx/config.json [hunk 2]
```diff
@@ -1,17 +1,17 @@
 {
   "$schema": "../schema.json",
-  "name": "Paperless-ngx",
+  "name": "Paperless (NGX)",
```
Comment (meienberger) [inline apps/paperless-ngx/config.json:3]: Same here, keep the current name

**Review by meienberger — CHANGES_REQUESTED** [review]

**Comment by steveiliop56** [comment]
@JigSawFr any updates?

**Comment by meienberger** [comment]
This was a nice work

**Pull request closed without merging**